test/extensions/filters/network/dubbo_proxy/dubbo_protocol_impl_test.cc

```diff
@@ -63,6 +65,68 @@ TEST(DubboProtocolImplTest, Normal) {
     EXPECT_EQ(1, context->bodySize());
     EXPECT_EQ(MessageType::Response, metadata->messageType());
   }
+
+  // Normal dubbo response with exception.
+  {
+    Buffer::OwnedImpl buffer;
+    Buffer::OwnedImpl body_buffer;
+    buffer.add(std::string({'\xda', '\xbb', 0x42, 20}));
+    addInt64(buffer, 1);
+
+    Hessian2::Encoder encoder(std::make_unique<BufferWriter>(body_buffer));
+    encoder.encode<int32_t>(0);
```

Comment:
Nit: I presume this line sets an exception in the response body. Could you add a comment about it?